src/ui/public/vislib/visualizations/_map.js

```diff
@@ -277,6 +280,10 @@ export default function MapFactory(Private) {
 
     // add map tiles layer, using the mapTiles object settings
     if (this._attr.wms && this._attr.wms.enabled) {
+      _.assign(mapOptions, {
+        minZoom: 1,
+        maxZoom: 18
+      });
       this._tileLayer = L.tileLayer.wms(this._attr.wms.url, this._attr.wms.options);
```

Comment:
Rather than override mapOptions, I wonder if this could go in this._attr.wms.options? It seems like this._attr.wms is where all the other wms overrides live.